src/app.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {
+  BrowserRouter as Router, Switch, Route, Link,
+} from 'react-router-dom';
+import BooksPage from './components/BooksPage';
+import Categories from './components/Categories';
+
+const App = () => (
+  <Router>
+    <nav>
+      <ul>
+        <li>
+          <Link to="/">Books</Link>
+        </li>
+        <li>
+          <Link to="/categories">Categories</Link>
+        </li>
+      </ul>
+    </nav>
+    <Switch>
+      <Route path="/categories">
+        <Categories />
+      </Route>
+      <Route path="/">
+        <BooksPage />
+      </Route>
+    </Switch>
```

Comment:
- [ ] _[OPTIONAL]_ Each route should have the attribute `exact` since `react-router` paths work with pattern matching(it will match against the value of `path`).